### DISCUSSION TITLE
Accessibility Audit Issues

### DISCUSSION THREAD

**Discussion by Studio88-KM** [code4recovery/12-step-meeting-list#952] [General]
Hey guys, like I've said so much, great job on this plugin and thank you so much for creating it and making it available to so many! 

I just had a web company who specializes in accessibility do an audit on a website that uses the 12 Meetings plugin and there were many issues. Would you be willing to look at the list and fix? 

Meeting search was confusing:

1. The meeting filters fields are not combined in a field set so it’s not obviously apparent that this search is distinguishable from the full site search in the header.
2. Status message is missing as the results change - nothing is announced to screen readers.
3. Dropdowns are coded as links not radio buttons (which is how they behave) - they are read out as links to the screen reader.
4. Make the number of results more meaningful. I.e., (7) should have an aria-label or hidden screen reader text so that it reads out (7 meetings) or (7 results).
5. When clicking on a meeting to get more information, the meeting loads in the same page and it’s not clear that the information loaded - needs an auditory announcement that the meeting information is loaded or focus to be shifted to the heading on that page.
6. There are elements in the meeting single (under the H3 for the days) that have been set up in lists that are not lists. This is confusing to screen reader users.
7. Maps are not helpful for screen reader users so the toggle to view the meeting map and maps on individual pages would be better hidden from screen readers with aria-hidden=”true”.
8. Allow the option for and H2

**Comment by joshreisner** [comment]
hi @Studio88-KM is this for a website using the TSML UI appearance or the Legacy UI?

**Reply by Studio88-KM** [reply]
Just added a #8. It's TSML UI. Look is much nicer.

**Reply by joshreisner** [reply]
Ok good. I will transfer this discussion to an issue on the TSML UI repo and follow up with you there.

https://github.com/code4recovery/tsml-ui/issues/263

**Reply by Studio88-KM** [reply]
Awesome, thank you!
